GetandTake/Services/Concrete/SupplierManager.cs

```diff
@@ -0,0 +1,50 @@
+﻿using GetandTake.DataAccess.Repositories.Abstract;
+using GetandTake.Models;
+using GetandTake.Services.Abstract;
+
+namespace GetandTake.Services.Concrete;
+
+public class SupplierManager : ISupplierService
+{
+    private readonly ISupplierRepository _repository;
+
+    public SupplierManager(ISupplierRepository repository)
+    {
+        _repository = repository;
+    }
+
+    public async Task CreateAsync(Supplier supplier)
+    {
+        await _repository.CreateAsync(supplier);
+    }
+
+    public void Delete(int supplierId)
+    {
+        _repository.Delete(entity => entity.SupplierID == supplierId);
+    }
+
+    public IEnumerable<Supplier> GetAll()
+    {
+        var suppliers = _repository.GetAll();
+
+        return suppliers;
```

Comment:
Guess you may use simple lambda expression here and for method below as well:
```suggestion
        public IEnumerable<Supplier> GetAll() =>
              _repository.GetAll();  
```